docs/general/ad-filtering/create-own-filters.md

```diff
@@ -3923,6 +3945,23 @@ domain.com##div.ad
 !#endif
 ```
 
+```adblock
+!#if (adguard && !adguard_ext_safari)
+! for all AdGuard products except AdGuard for Safari
+||example.org^$third-party
+domain.com##div.ad
+!#else
+! for AdGuard for Safari only
+||subdomain.example.org^$third-party
+!#endif
+```
+
+:::info Compatibility
+
+The `!#else` directive is supported by the [FiltersDownloader][gh-filters-downloader] v1.1.20 or later.
```

Comment:
We should probably explain what it means, i.e. that filter lists that are built by FiltersRegistry will apply `#else`, and when you add custom lists to AG it won't be supported **yet** (until CoreLibs and tsurlfilter supports it).

Reply:
tsurlfilter has nothing to do with it, FiltersDownloader is used by the browser extension directly. still worth mentioning the versions of extension and corelibs, agree

Reply:
Makes sense, but I guess we should mention at least major releases of our products in which it will be used.

For instance, say this directive is planned to be supported in the AG extensions v4.3 or newer, and products based on CoreLibs v1.14.